db/migrate/20241029125406_add_reference_to_security_prices.rb

```diff
@@ -0,0 +1,5 @@
+class AddReferenceToSecurityPrices < ActiveRecord::Migration[7.2]
+  def change
+    add_reference :security_prices, :security, foreign_key: true, type: :uuid
```

Comment:
I'd also remove `ticker` from the `security_prices` table to avoid any confusion.  We'll also want to link any existing prices to their relevant security; something like (pseudocode below):

```rb
Security::Price.each do |sp|
  security = Security.find_by(ticker: sp.ticker)
  sp.security = security
  sp.save!
end
```

Reply:
Good call on linking. I'm goign to save removing the ticker until after this is deployed/migrated so we have the ticker data in case something goes wrong with the data migration.